src/streamlit_analytics2/main.py

```diff
@@ -279,31 +417,33 @@ def start_tracking(
     load_from_json: Optional[Union[str, Path]] = None,
     streamlit_secrets_firestore_key: Optional[str] = None,
     firestore_project_name: Optional[str] = None,
+    session_id: Optional[str] = None,
 ):
     """
     Start tracking user inputs to a streamlit app.
-
```

Comment:
Has this changed? Is `.stop_tracking()` no longer required?

Reply:
Nope, I believe this is my over-eager Cursor IDE autocomplete causing havoc. Put back the missing docstring (I assume that's what you're referring to?)